src/unicodeSymbols.js

```diff
@@ -35,23 +38,27 @@ export default {
     "\u1e17": "\u0065\u0304\u0301",  // ḗ = \=\'{e}
     "\u1e15": "\u0065\u0304\u0300",  // ḕ = \=\`{e}
     "\u0115": "\u0065\u0306",  // ĕ = \u{e}
+    "\u1e1d": "\u0065\u0306\u0327",  // ḝ = \u\c{e}
     "\u011b": "\u0065\u030c",  // ě = \v{e}
     "\u00ea": "\u0065\u0302",  // ê = \^{e}
     "\u1ebf": "\u0065\u0302\u0301",  // ế = \^\'{e}
     "\u1ec1": "\u0065\u0302\u0300",  // ề = \^\`{e}
     "\u1ec5": "\u0065\u0302\u0303",  // ễ = \^\~{e}
     "\u0117": "\u0065\u0307",  // ė = \.{e}
+    "\u0229": "\u0065\u0327",  // ȩ = \c{e}
     "\u1e1f": "\u0066\u0307",  // ḟ = \.{f}
     "\u01f5": "\u0067\u0301",  // ǵ = \'{g}
     "\u1e21": "\u0067\u0304",  // ḡ = \={g}
     "\u011f": "\u0067\u0306",  // ğ = \u{g}
     "\u01e7": "\u0067\u030c",  // ǧ = \v{g}
     "\u011d": "\u0067\u0302",  // ĝ = \^{g}
     "\u0121": "\u0067\u0307",  // ġ = \.{g}
+    "\u0123": "\u0067\u0327",  // ģ = \c{g}
```

Comment:
This one is not going to render correctly (I assume); see @ylemkimon's [recent comment](https://github.com/KaTeX/katex-fonts/pull/26#issuecomment-362148045).  Perhaps we should fix this in another PR, though.  (It'll need another tweak to the font, I suspect.)

Reply:
I'm not sure what the resolution is for `\c{g}`.  Let's handle it separately.